pystac/catalog.py

```diff
@@ -280,11 +284,11 @@ def add_item(
         item: Item,
         title: Optional[str] = None,
         strategy: Optional[HrefLayoutStrategy] = None,
-        keep_parent: bool = False,
+        set_parent: bool = True,
     ) -> None:
         """Adds a link to an :class:`~pystac.Item`.
-        This method will set the item's parent to this object (except if a parent
-        is set and keep_parent is true).
+        This method will set the item's parent to this object (unless ``set_parent`` is
```

Comment:
Should we mention here that the self link may also get updated if the parent it set?